drivers/dma/dma_nxp_sdma.c

```diff
@@ -395,6 +423,12 @@ static int dma_nxp_sdma_reload(const struct device *dev, uint32_t channel, uint3
 		return 0;
 	}
 
+	/* allow reload only for active channels */
+	if (chan_data->state != SDMA_CHAN_STATE_STARTED) {
+		LOG_ERR("%s: invalid state %d", __func__, chan_data->state);
```

Comment:
drop the function name prefix. `CONFIG_LOG_FUNC_NAME_PREFIX_` should already do the job.